colaraz/cms/static/js/header.js

```diff
@@ -74,45 +78,119 @@ function getAndPopulateNotifications() {
     });
 
     function handleNotificationsListing(notifications) {
-        $("#notifications-list").html(notifications);
+        $('#notifications-list').html(notifications);
     }
     function handleNotificationsCount(count) {
         if (count == 0) {
-            let countSpan = $("#unread-notifications-count");
-            countSpan.removeClass("count");
-            countSpan.html("");
+            let countSpan = $('#unread-notifications-count');
+            countSpan.removeClass('count');
+            countSpan.html('');
         } else {
-            let countSpan = $("#unread-notifications-count");
-            countSpan.addClass("count");
+            let countSpan = $('#unread-notifications-count');
+            countSpan.addClass('count');
             countSpan.html(count);
         }
     }
 
     function createNotification(img_src, description, days_count, read) {
-        return `<li class="${read ? 'read' : 'unread'}" >` +
-            `<div class="media">` +
-            `<a href="#"><img src="${img_src}" alt=""></a>` +
-            `</div>` +
-            `<div class="description">` +
-            `${description}` +
-            `<p>${days_count}</p>` +
-            `</div>` +
-            `</li>`;
+        return `<li class='${read ? 'read' : 'unread'}' >
+                    <div class='media'>
+                        <a href='#'>
+                            <img src='${img_src}' alt=''>
+                        </a>
+                    </div>
+                    <div class='description'>
+                        ${description}
+                        <p>${days_count}</p>
+                    </div>
+                </li>`;
     }
 }
 
 function markNotificationsAsRead() {
     $.ajax({
-        type: "GET",
+        type: 'GET',
         url: colarazNotificationsMarkingUrl
     });
 }
 
 function removeNotificationsCount() {
-    let countSpan = $("#unread-notifications-count");
-    if (countSpan.html() !== "" || countSpan.hasClass("count")) {
-        countSpan.removeClass("count");
-        countSpan.html("");
+    let countSpan = $('#unread-notifications-count');
+    if (countSpan.html() !== '' || countSpan.hasClass('count')) {
+        countSpan.removeClass('count');
+        countSpan.html('');
         markNotificationsAsRead();
     }
 }
+
+function getAndPopulateJobAlerts() {
+    $.ajax({
+        type: 'GET',
+        url: colarazJobAlertsFetchingUrl,
+        success: function (resp) {
+            let job_alerts = '';
```

Comment:
Please let's use camelCase naming notation where ever we can
```suggestion
            let jobAlerts = '';
```

Reply:
done